src/hazelcore/dynamics/DHExp.re

```diff
@@ -82,6 +82,36 @@ module BinIntOp = {
     };
 };
 
+module UnIntOp = {
+  [@deriving sexp]
+  type t =
+    | Negate;
+  let of_op = (op: UHExp.unop): option((t, HTyp.t)) =>
+    switch (op) {
+    | Negate => Some((Negate, Int))
+    | _ => None
+    };
+  let to_op = (bio: t): UHExp.unop =>
```

Comment:
style: `bio` isn't a good variable name here (i think it stood for "bin int op")